ivy/src/main/scala/sbt/internal/librarymanagement/ConvertResolver.scala

```diff
@@ -406,4 +407,14 @@ private[sbt] object ConvertResolver {
       }
     }
   }
+
+  private sealed trait ThreadSafeSshBasedResolver
+      extends org.apache.ivy.plugins.resolver.AbstractSshBasedResolver {
+//    private def lock = new Object
```

Comment:
commented out code

Reply:
do you want me to remove it? i left it to easily test the changes

Reply:
oh right it was intentional. could you add a little comment please?

Reply:
added comments